docker/Dockerfile

```
@@ -76,10 +87,19 @@ LABEL vsc-ref=${GIT_COMMIT}
 
 WORKDIR /
 COPY --from=builder /workspace/gpu-operator /usr/bin/
+COPY --from=builder /workspace/validator /usr/bin/nvidia-validator
```

nit: let's create a follow-up to also rename the actual executable.

---

Let's name it either gpu-operator-validator or operator-validator
```suggestion
COPY --from=builder /workspace/validator /usr/bin/operator-validator
```

---

I have updated the binary name to `nvidia-validator` following a discussion with @tariq1890